src/armory/Armory.js

```diff
@@ -100,6 +100,10 @@ class Armory extends React.Component<Props, State> {
 	getTanks(): void {
 		getAllUsersTanks((successful, allTanks) => {
 			if (successful) {
+				if (allTanks.length === 0) {
+					console.log('Expected to have at least one tank!');
+					return;
+				}
```

Comment:
Did you just place this in since it was getting annoying entering the Armory if you didn't own a tank? That's the only thing that I can think of.

Reply:
Yeah, I was getting an error and it was confusing. I am pretty sure it was just because my database was screwed up though, it wasn't actually anything wrong with the code